lib/hammer_cli_foreman/host.rb

```diff
@@ -413,6 +413,22 @@ class RebuildConfigCommand < HammerCLIForeman::SingleResourceCommand
       build_options
     end
 
+    class ENCDump < HammerCLIForeman::SingleResourceCommand
+      action :enc
+
+      command_name "enc-dump"
+      desc _("Dump host's ENC YAML")
+      failure_message _("Cannot retrieve ENC values of host")
+
+      def execute
+        data = send_request
+        print_message(YAML.dump(data))
```

Comment:
Since the ENC is always in YAML and we don't support different output per output adapter yet, I'd use just `puts` instead of `print_message`. It doesn't play well with other output adapters:
```
$ hammer --output=json host enc-dump --name test3.tstracho-laptop                                                                 
{
  "message": "---\nparameters:\n  kt_env: \n  kt_cv: \n  foreman_host_collections: []\n  lifecycle_environment: \n  content_view: \n  content_view_info: {}\n  hostgroup: parent/Test\n  foreman_subnets:\n  - name: tstracho-laptop\n    network: 192.168.122.0

$ hammer --output=yaml host enc-dump --name test3.tstracho-laptop                                                                 
---
:message: "---\nparameters:\n  kt_env: \n  kt_cv: \n  foreman_host_collections: []\n
  \ lifecycle_environment: \n  content_view: \n  content_view_info: {}\n  hostgroup:
  parent/Test\n  foreman_subnets:\n  - name: tstracho-laptop\n    network: 192.168.122.0\n
  \   mask: 255.255.255.0\n    gateway: ''\n    dns_primary: 192.168.122.1\n    dns_secondary:
  8.8.8.8\n 
```
(note the "message" key)